### PR TITLE
lintTree hook

### LINKED ISSUE #124: `lintTree` hook is not yet implemented

### PR DESCRIPTION
**Pull request by t-sauer**
Closes #124

### CODE REVIEW

File: lib/broccoli/glimmer-app.ts [hunk 1]
```diff
@@ -364,6 +381,23 @@ Please run the following to resolve this warning:
...
+
+    let templateTree = new Funnel(src, {
+      include: ['**/*.hbs']
+    });
+
+    let srcTree = new Funnel(src, {
+      exclude: ['**/*.hbs']
+    });
+
+    let lintedTemplates = addonLintTree(this.project, 'templates', templateTree);
```
Comment (rwjblue) [inline lib/broccoli/glimmer-app.ts]: I _vaguely_ recall thus being singular in Ember CLI’s pipeline, can you double check?

Comment (t-sauer) [reply lib/broccoli/glimmer-app.ts]: Just rechecked: It is also `templates` in Ember CLI (https://github.com/ember-cli/ember-cli/blob/35536c7c378751a406766e836081aa47971d250b/lib/broccoli/ember-app.js#L1346)

Comment (rwjblue) [reply lib/broccoli/glimmer-app.ts]: Awesome, thank you (and sorry for the run around)...

**Comment by acorncom** [comment]
@rwjblue is this ready to merge then?

**Comment by rwjblue** [comment]
@glimmer/application-pipeline 0.9.1 published 🎉

**Pull request merged**